### ISSUE TITLE
Does this work with current firmware?

### ISSUE THREAD

**Issue by Thomasbomb** [valkjsaaa/node-switchmate3#8]
I am considering buying a switchmate, but I won't pull the trigger until I can find a github repo that lets me control it without a hub

**Comment by valkjsaaa** [comment]
Yes. It works with the latest firmware, although you should NOT expect the same level of responsiveness and robustness as a WiFi switch.

**Comment by Thomasbomb** [comment]
Ok, thanks for the response. I'm ok with delays of a second or two, but the robustness issue concerns me slightly. Can you comment on how reliable it is?
